cfgov/jinja2/v1/owning-a-home/oah-email-signup.html

```diff
@@ -0,0 +1,73 @@
+
+{# ==========================================================================
+
+   render(links)
+
+   ==========================================================================
+
+   Description:
+
+   Renders all the brand footer content: 3 link sections and email section.
+
+   links: An array of three link objects, each containing:
+            - text (req): text of link section
+            - img: img for link section
+            - label: link label
+            - url: link url
+
+            Macro will display max of three link sections. If there are
+            more than three link objects, only the first three will
+            be shown. If no links are passed in, only the email signup
```

Comment:
Same comment about the email signup here.

Reply:
In fact, why are we duplicating these templates? If they have different classes could we pass those as an argument?

Reply:
Due to the slight differences, I've created as separate module for now. Think of OAH as a separate app for now.